crates/adapters/okx/Cargo.toml

```diff
@@ -83,6 +83,7 @@ nautilus-testkit = { workspace = true }
 criterion = { workspace = true }
 rstest = { workspace = true }
 tracing-test = { workspace = true }
+url = "2.5"
```

Comment:
Do we definitely need a new specific crate dependency? there could be a crate in the workspace we're already using for this.